src/content/docs/docs/intro/web-monetization-extension.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: Web Monetization extension
+---
+
+import { Steps } from '@astrojs/starlight/components';
+import { LinkOut } from "@interledger/docs-design-system";
+
+The Interledger Foundation's Web Monetization extension allows you to support your favorite web monetized sites directly from your web browser in a quick, non-intrusive, and privacy-preserving way.
+
+### Web monetized sites
+
+To use the extension to make Web Monetization payments, the site you visit must be web monetized. Since Web Monetization is fairly new, we encourage content owners and publishers tell their audience they're web monetized via a banner or some other method.
+
+If you're interested in web monetizing your own content, the [Add Web Monetization to a page](/docs/guides/monetize-page) guide is a good place to start.
+
+### Web Monetization payments 
+
+To make Web Monetization payments, you only need an account with a [Web Monetization-enabled wallet provider](/docs/resources/op-wallets). There's no site-by-site account or subscription required for each individual site you want to support.
+
+A web monetized site cannot initiate payments from your digital wallet. All payments are explicitly initialized and authorized by you. The sites you choose to pay are not provided with any of your personally identifiable information, such as your name, email address, or IP address. 
```

Comment:
To be clear, there's not a choice exactly to pay or not pay given site at the moment - there's only global enable/disable, not per site.

Also, I think "All payments are explicitly initialized" might give impression user has to initiate payments and not it happens behind the scenes automatically?